packages/rolldown/src/plugin/builtin-plugin.ts

```diff
@@ -68,6 +77,10 @@ export function wasmPlugin() {
   return new WasmPlugin()
 }
 
+export function ecmaTransformPlugin(config?: TransformPluginConfig) {
```

Comment:
```suggestion
export function transformPlugin(config?: TransformPluginConfig) {
```

I think you could just call it `transform` and don't need to worried about if it's conflicted if we add css, other language support.

Reply:
Just like the minifier and other config. 

`Input#entryFilenames` `Output#minifier` is for JS.
`Input#cssEntryFilenames` `Output#cssMinifier` for css.

We could use non-prefix word to represent config related to JS, since it's most common part of bundler.

Reply:
Better add some jsdoc on this function and saying that in this plugin `jsx`, `ts`, `tsx` would be transformed.